### PR TITLE
Migrate tab sheet to use BrowserStore instead of SessionManager.

### PR DESCRIPTION
**Pull request by pocmo**

### CODE REVIEW

**Comment by pocmo** [comment]
```
> Could not resolve all files for configuration ':app:lintClassPath'.
   > Could not download groovy-all.jar (org.codehaus.groovy:groovy-all:2.4.15)
      > Could not get resource 'https://jcenter.bintray.com/org/codehaus/groovy/groovy-all/2.4.15/groovy-all-2.4.15.jar'.
         > Could not GET 'https://jcenter.bintray.com/org/codehaus/groovy/groovy-all/2.4.15/groovy-all-2.4.15.jar'.
            > Connection reset
```

Let's try again...

File: app/src/main/java/org/mozilla/focus/session/ui/TabViewHolder.kt [hunk 1]
```diff
@@ -6,40 +6,39 @@ package org.mozilla.focus.session.ui
...
     private val textView: TextView
 ) : RecyclerView.ViewHolder(textView), View.OnClickListener {
     companion object {
-        @JvmField
-        internal val LAYOUT_ID = R.layout.item_session
+        internal const val LAYOUT_ID = R.layout.item_session
     }
 
-    private var sessionReference: WeakReference<Session> = WeakReference<Session>(null)
+    private var tabReference: WeakReference<TabSessionState> = WeakReference<TabSessionState>(null)
```
Comment (jonalmeida) [inline app/src/main/java/org/mozilla/focus/session/ui/TabViewHolder.kt:27]: nit: TabSessionState probably doesn't need to be a weakref any more.

**Review by jonalmeida — APPROVED** [review]
📑 ✅ ✨

File: app/src/main/java/org/mozilla/focus/session/ui/TabViewHolder.kt (hunk 1 above)
Comment (pocmo) [reply app/src/main/java/org/mozilla/focus/session/ui/TabViewHolder.kt:27]: Oh, yeah, right. I didn't even process that code. Looks like we only keep the tab around to select it later, and we can do that with just the tab ID now, so no need to keep more than that. :)

**Pull request merged**